.github/workflows/post-merge.yaml

```diff
@@ -57,10 +57,57 @@ jobs:
         run: make ci-go-test-coverage
         timeout-minutes: 30
 
+  release-build:
+    name: Release Build (linux, windows)
+    runs-on: ubuntu-18.04
+    needs: generate
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Build Linux and Windows
+        run: make ci-go-build-linux ci-go-build-windows
+        timeout-minutes: 30
+        env:
+          TELEMETRY_URL: ${{ secrets.TELEMETRY_URL }}
+
+      - name: Upload binaries
+        uses: actions/upload-artifact@v2
+        if: always()
+        with:
+          name: binaries
+          path: _release
+
+  release-build-darwin:
+    name: Release Build (darwin)
+    runs-on: macos-latest
```

Comment:
👉 https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on

`macos-latest` is 10.15, not big sur. But the binaries work on the latter, too.